tests/test_metrics_quantiles.py

```diff
@@ -0,0 +1,77 @@
+"""
+Tests for quantile metrics.
+"""
+
+import pytest
+import numpy as np
+
+from uncertainty_toolbox.metrics_quantile_models import *
```

Comment:
For consistency with the other test files, could you import each function instead of using *.